apiserver/facades/client/client/status.go

```diff
@@ -133,12 +133,9 @@ func (c *Client) StatusHistory(request params.StatusHistoryRequests) params.Stat
 			continue
 		}
 
-		var (
-			err  error
-			hist []params.DetailedStatus
-		)
+		var err error
```

Comment:
This change is a matter of taste I would have thought? I prefer a bunch of vars to be grouped together. YMMV

Reply:
ah, ok, i'll re group, don't know why it didn't like that